actor/builtin/miner/miner.go

```diff
@@ -460,12 +462,12 @@ func (ma *Actor) CommitSector(ctx exec.VMContext, sectorID uint64, commD, commR,
 		// the PoRep verification operation needs to know some things (e.g. size)
 		// about the sector for which the proof was generated in order to verify.
 		//
-		// It is undefined behavior for a miner using "LiveMode" to verify a
-		// proof created by a miner using "TestMode" (and vice-versa).
+		// It is undefined behavior for a miner using "LiveMode" to verify a proof
+		// created by a miner in "TestMode"(and vice-versa).
 		//
-		proofsMode := proofs.LiveMode
-		if os.Getenv("FIL_USE_SMALL_SECTORS") == "true" {
-			proofsMode = proofs.TestMode
+		proofsMode, err := getProofsMode(ctx)
+		if err != nil {
+			return ErrGetProofsModeFailed, Errors[ErrGetProofsModeFailed]
 		}
 
 		req := proofs.VerifySealRequest{}
```

Comment:
Non-blocking. This code to `VerifySeal` really shouldn't live here. I've created an issue for it. #2555